src/vma/netlink/netlink_compatibility.cpp

```diff
@@ -112,13 +112,26 @@ int nl_cache_mngr_compatible_add(struct nl_cache_mngr*	mngr, const char* name, c
 	return err;
 }
 
-int nl_object_get_compatible_msgtype(const struct nl_object* obj) {
-	return nl_object_get_msgtype(obj);
+in_addr_t nl_object_get_compatible_gateway(struct rtnl_route* nl_route_obj) {
```

Comment:
since it was called only in netlink_event(struct nl_object* obj, void* notifier) constructor which is removed cause of not usage.